src/lib/swapper/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -166,12 +163,14 @@ export const getL1quote = async (
   }
 
   const getRouteBuyAmount = (quote: ThornodeQuoteResponseSuccess) => {
-    const emitAsset = bn(quote.expected_amount_out).plus(quote.fees.outbound)
+    const emitAsset = bn(quote.expected_amount_out).plus(quote.fees.total)
     return toBaseUnit(fromBaseUnit(emitAsset, THORCHAIN_FIXED_PRECISION), buyAsset.precision)
   }
 
   const getProtocolFees = (quote: ThornodeQuoteResponseSuccess) => {
-    const buyAssetTradeFeeBuyAssetCryptoThorPrecision = bnOrZero(quote.fees.outbound)
+    const buyAssetTradeFeeBuyAssetCryptoThorPrecision = bnOrZero(quote.fees.total).minus(
+      quote.fees.affiliate,
+    )
```

Comment:
@woodenfurniture not blocking the PR for that, but could we make things more declarative and commented in code, to keep things context-free? 

i.e

1. add a comment in code that does not explain *what* the protocol fees are (what this comment does), but rather *why* the affiliate isn't part of the protocol fee, and what could happen if we didn't do so i.e the bug we're experiencing currently
2. instead of doing total fees - affiliate fees, being explicit and doing liquidity fees + outbound fees, which will yield the same amount, but makes the above clear 

Something like:

```suggestion
    // THORChain fees consist of liquidity, outbound, and affiliate fees
    // For the purpose of displaying protocol fees to the user, we don't need the latter
    // The reason for that is the affiliate fee is shown as its own "ShapeShift fee" section
    // Including the affiliate fee here would result in the protocol fee being wrong, as affiliate fees would be
    // double accounted for both in protocol fees, and affiliate fee
    const buyAssetTradeFeeBuyAssetCryptoThorPrecision = bnOrZero(quote.fees.liquidity).plus(
      quote.fees.outbound,
    )
```

Reply:
Comment added in c2aab5e4f2ab05beee8ebdca421642080a74289d - ditched the declarative version of this because of risks calc-wise